### DISCUSSION TITLE
Remove ^ref from file

### DISCUSSION THREAD

**Discussion by mvm-sylverio** [hadynz/obsidian-kindle-plugin#219] [Q&A]
Is it possible to remove the ^ref from each highlight?

![image](https://user-images.githubusercontent.com/97754180/201435751-2c1c2ef9-6bf8-4b7e-b03b-a295182b7916.png)

**Comment by hadynz** [comment]
`^ref-` is used by the plugin to identify your highlight in order to re/sync. The plugin doesn't have an option to remove it at the moment.

**Comment by TomYeoman** [comment]
It would be a nice README piece to mention that's what the ref is, had to look around a bit
